addon/components/ember-popper-base.js

```diff
@@ -145,7 +145,12 @@ export default class EmberPopperBase extends Component {
   // ================== LIFECYCLE HOOKS ==================
 
   didUpdateAttrs() {
+    if (this._updateRAF !== null) {
```

Comment:
Partially added this check to make sure we only schedule once for the observers, but it's good to do anyways since there are no strong guarantees about RAF and runloop updates. Theoretically, `didUpdateAttrs` could be called more than once before the RAF has occured.